functions/InsertBookingSlotFromQueue/project.json

```diff
@@ -0,0 +1,14 @@
+{
+    "frameworks": {
+        "net46":{
+            "dependencies": {
+                "Microsoft.IdentityModel.Clients.ActiveDirectory": "2.16.204221202", 
+                "Microsoft.Azure.KeyVault": "1.0.0",
```

Comment:
The booking does use encryption, but that is now in a util method (EncryptionUtils.csx). You are right, this is where I have created the function from a copy. Is there a way to "Remove unused references" like you would in full fat vs? I tried to build up the references initially, but it was painful, so I just copied in all the usings (lazy sorry!)